src/components/layout/CardTitle.tsx

```diff
@@ -15,16 +23,28 @@ const CardTitle = function CardTitle({
   classes,
   elementRef,
 
+  tagName = 'h1',
+  variant = 'primary',
+
   ...htmlAttributes
 }: CardTitleProps) {
+  const WrapperElement = tagName;
   return (
     <div
       data-component="CardTitle"
       {...htmlAttributes}
-      className={classnames('grow text-lg text-brand font-semibold', classes)}
+      className={classnames(
+        {
+          'text-lg text-brand font-semibold': variant === 'primary',
+          'text-xl text-slate-7 font-normal': variant === 'secondary',
```

Comment:
Is the text supposed to be bigger on `secondary` variant? Is that intended?

Reply:
Yes, it actually is!